app/modules/import-export/module.php

```diff
@@ -762,14 +762,14 @@ private function get_summary_titles() {
 				// CPT data appears in two arrays:
 				// 1. content object: in order to show the export summary when completed in getLabel function
 				$summary_titles['content'][ $custom_post_type ] = [
-					'single' => $custom_post_types_object->labels->singular_name,
-					'plural' => $custom_post_types_object->label,
+					'single' => ! empty( $custom_post_types_object ) ? $custom_post_types_object->labels->singular_name : '',
+					'plural' => ! empty( $custom_post_types_object ) ? $custom_post_types_object->label : '',
```

Comment:
```suggestion
					'plural' => custom_post_types_object->label  ?? '',
```

Reply:
To clarify - the code on line 765/766 might still fail if $custom_post_types_object exists but label does not exit. I'd suggest checking on the actual object and key to be used just to be on the safe side.

Reply:
Update.